apps/projects/api.py

```diff
@@ -30,12 +29,8 @@ def get_queryset(self):
 
 
 class ModerationProjectsViewSet(viewsets.ReadOnlyModelViewSet):
-    serializer_class = ProjectSerializer
+    serializer_class = ModeratorProjectSerializer
     permission_classes = (permissions.IsAuthenticated,)
 
     def get_queryset(self):
-        project_report_count = {
-            project: helpers.get_num_reports(project)
```

Comment:
from what I understand this doesn't actually work? the number of reports isn't sent as the serializer drops it. I moved it in the serializer and added the sorting to the front end

Reply:
it also seems to be in helpers and api? but that's all i know

Reply:
Should we for the sake of speeding it up just sort by something we can more easily get for now? Like `-created`